src/Strategy/CommonClassesStrategy.php

```diff
@@ -77,6 +78,10 @@
             ['class' => Cake::class, 'condition' => Cake::class],
             ['class' => Zend::class, 'condition' => Zend::class],
             ['class' => Artax::class, 'condition' => Artax::class],
+            [
+                'class' => [self::class, 'buzzInstantiate'],
+                'condition' => [\Buzz\Client\FileGetContents::class, \Buzz\Message\ResponseBuilder::class],
```

Comment:
are those in separate packages? or is it a bit of a hack to see if a new enough version of buzz is available?

Reply:
It is a version hack =)